scalafix-core/shared/src/main/scala/scalafix/internal/rule/DisableVariantTypes.scala

```diff
@@ -0,0 +1,26 @@
+package scalafix.internal.rule
+
+import scalafix.rule.RuleCtx
+import scalafix.{LintMessage, SemanticRule, SemanticdbIndex}
+import scala.meta._
+import scalafix.lint.LintCategory
+
+final case class DisableVariantTypes(index: SemanticdbIndex)
```

Comment:
I propose we make this a part of `DisableSyntax.contravariantType/covariantType: Boolean`

Reply:
Goog idea!